.github/workflows/main.yml

```diff
@@ -16,8 +16,9 @@ on:
 
 env:
   IMAGE: frontstage
-  REGISTRY_HOSTNAME: eu.gcr.io
+  REGISTRY_HOSTNAME: europe-west2-docker.pkg.dev
   HOST: ${{ secrets.GOOGLE_PROJECT_ID }}
+  GAR_REPOSITORY: ${{ secrets.GAR_REPOSITORY }}
```

Comment:
I appreciate we've copied the previous migrations, but unless `secret.GAR_REPOSITORY` will resolve to different values per repo we should probably not make this a 'secret'. I think we should make it `images` explicitly in the env: vars and do that for all upcoming migrations too.

Reply:
Good point, I've updated it